ansible/roles/test/tasks/sonic.yml

```diff
@@ -116,6 +116,10 @@
   include: mtu.yml
   tags: mtu
 
+- name: Directed Broadcast test
```

Comment:
Can you also adjust the comment on lines 109-110 to refer to "the following tests" (similar to the comment on lines 7-9)? You can then also remove the comment from lines 123-124, as they will then be redundant.

Reply:
Done.